readme.md

```diff
@@ -47,7 +47,10 @@ RealWorld.io frontend.
 
 # Getting started
 
-You need to have docker, [pipenv](https://pipenv.readthedocs.io/) and Python 3.8 installed on your machine. Docker should be running. Then you can run:
+You need to have [Docker](https://www.docker.com), [Nix](https://nixos.org/nix) and [direnv](https://direnv.net/) installed and configured on your system.
```

Comment:
Nix and direnv should be optional. The Readme should first give instructions how to install dependencies on Ubuntu and MacOS (using `apt-get` and `brew`) and finally tell the user about nix, as a B-plan if they have problems.